xplique/plots/metrics.py

```diff
@@ -70,8 +70,11 @@ def barplot(
             # methods_color is a string linking to a cmap
             cmap = matplotlib.cm.get_cmap(methods_colors)
 
-        methods_colors = {methods[i]: cmap(i / (len(methods) - 1))
-                          for i in range(len(methods))}
+        if len(methods) == 1:
```

Comment:
if methods is a base explainer it has no len ^^
Also if len(methods) == 1 then the second line will work ;)

Reply:
corrected now